### DISCUSSION TITLE
folder icon thumbnail

### DISCUSSION THREAD

**Discussion by Red0Hood** [spacedriveapp/spacedrive#1586] [Feature Request]
### just like windows folder where if you have desktop.ini and icon you get a thumbnail
- while spacedrive indexing folders for files check if the folder have desktop.ini then display the icon

**Comment by utkubakir** [comment]
Can you send some examples?

**Reply by Red0Hood** [reply]
## in the image below thumbnail of folder
- in windows you make them by making desktop.ini and icon
- this is what is inside desktop.ini
``` html
[.ShellClassInfo]
IconResource= ./A Bit Of Fry & Laurie.ico,0
[ViewState]
Mode=
Vid=CLSID2
IconFile = ./A Bit Of Fry & Laurie.ico
FolderType=Videos
```
![1](https://github.com/spacedriveapp/spacedrive/assets/30514412/fa165dfd-3287-43c1-81d2-fa6bc163184b)
![2](https://github.com/spacedriveapp/spacedrive/assets/30514412/7da4e092-6e7f-4e40-aca0-e21e03ab0e46)
